### ISSUE TITLE
Image elements do not have [alt] attributes

### ISSUE THREAD

**Issue by Granoffel** [hochschule-darmstadt/einander-helfen#468]
**Reason (Why?)**
Provide an alt attribute for every <img> element. If the image fails to load, the alt text is used as a placeholder so users have a sense of what the image was trying to convey. Also an alt tag is important for crawler to list the image.

**Solution (What?)**
Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute
